assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -93,3 +93,16 @@ a.inverted {
   }
 }
 
+.signup-social {
+
+  h3.ui.header {
+    color: white;
+    margin-bottom: @h3;
+    text-align: center;
+  }
+}
+
+#signup-orcid-logo {
```

Comment:
Ideally this should replicate semantic-ui `icon`, to be similar to SUI github's icon. I tried to use variables defined in SUI less but with no success.

Let me know if this can be implemented in relation to SUI `icon`